app/blueprints/user_blueprint.rb

```diff
@@ -43,5 +43,9 @@ class UserBlueprint < Blueprinter::Base
     field :invited_by_email do |user, _options|
       user.invited_by&.email
     end
+
+    association :invited_to_jurisdiction, blueprint: JurisdictionBlueprint, view: :minimal do |user, _options|
+      user.jurisdiction_memberships.order(updated_at: :desc).first.jurisdiction
```

Comment:
Shouldn't this have optional chaining?

`user.jurisdiction_memberships.order(updated_at: :desc).first&.jurisdiction` because there might be no memberships

Reply:
ah! good catch!